### PR TITLE
Fixed issue with ORDER BY several columns and IN / NOT IN operations with String / Number objects

### PR DESCRIPTION
**Pull request by akhaneev**
Two fixes for String / Number objects:

1. ORDER BY two or more columns

1. IN / NOT IN operators

tests811 has two new tests #5 and #6. Other changes is something with formatting.
test813 is ignored - tests for issues #1263 and #1264

### CODE REVIEW

File: src/15utility.js [hunk 1]
```diff
@@ -1082,6 +1082,13 @@ var extend = (utils.extend = function extend(a, b) {
 	return a;
 });
 
+/**
+ * Extracts the primitive data
+ */
+var getValueOf = (utils.getValueOf = function (val) {
```
Comment (mathiasrw) [inline src/15utility.js:1088]: How come we need to have a new way of accessing the value?

**Review by mathiasrw — COMMENTED** [review]
Thank you so much for taking the time to look into this. I dont quite sure getting the valueOf is the best way on changing how IN and ORDER BY works. Can you please reflect on why you have done it this way?

**Comment by mathiasrw** [comment]
Please run `yarn format` to fix tests.

File: src/15utility.js (hunk 1 above)
Comment (akhaneev) [reply src/15utility.js:1088]: String and Number objects can't be compared the same way as primitives. So we need to convert them to primitives with valueOf() before comparing.

**Comment by akhaneev** [comment]
`yarn format` fails to run, it says `{` is not a command:
`$ yarn pretty-since-dev --write`
`$ { git diff --name-only --diff-filter=d origin/develop ; git diff --name-only --diff-filter=d --staged origin/develop ; } | sort | uniq | grep -vE '^dist/|^test/coverage|^lib/|.min.js$' | grep -E '\.(scss|css|js|ts|vue|json)$' | xargs npx prettier`
` --write`
`"{" is not a command`
`error Command failed with exit code 255.`

**Comment by mathiasrw** [comment]
Ahh - You must be on windows. ill PR your branch.

**Comment by akhaneev** [comment]
Yes, on Windows. Thanks!

**Comment by mathiasrw** [comment]
Please try to run `npx prettier --write 'src/*.js' 'test/*.js'` and repush.

**Comment by akhaneev** [comment]
Thank you! Done.

**Comment by akhaneev** [comment]
test621 runs fine in my environment...

**Comment by mathiasrw** [comment]
Yep. Its just a timeout. Running them again...

**Comment by akhaneev** [comment]
Hi Mathias,
test/test813.js was not modified. I fixed it earlier when merged with the upstream.

File: src/55functions.js [hunk 2]
```diff
@@ -278,7 +278,10 @@ stdlib.UPPER = stdlib.UCASE = function (s) {
 // Concatination of strings
 stdfn.CONCAT_WS = function () {
 	var args = Array.prototype.slice.call(arguments);
-	return args.slice(1, args.length).join(args[0]);
+	args = args.filter(function(el, i){
+		return !i || (el !== null && el !== undefined);
```
Comment (mathiasrw) [inline src/55functions.js:282]: What is happening here?

**Comment by mathiasrw** [comment]
Lets sort this out and get it merged.

**Comment by mathiasrw** [comment]
Hmm. Still not able to get that test nailed.

**Comment by mathiasrw** [comment]
Argh. I fucked up. Working on it...

**Comment by mathiasrw** [comment]
Released as part of v2.1.0

**Pull request closed without merging**